src/world/area_omo/omo_03/omo_03_2_main.c

```diff
@@ -50,7 +50,7 @@ EvtScript N(EVS_Main) = {
 #if VERSION_PAL
     EVT_CALL(GetLanguage, LVar0)
     EVT_SWITCH(LVar0)
-        EVT_IF_GE(LVar0, 2)
+        EVT_IF_GE(LVar0, LANGUAGE_FR) // or LANGUAGE_ES
             EVT_SUB(LVar0, 2)
```

Comment:
Would be cool to know what the STATION signs look like in other languages. It seems like English is the same as French and German the same as Spanish?

Reply:
Turns out both German and Spanish just swap to generic symbols.

![image](https://user-images.githubusercontent.com/737603/227779490-55aadb86-4604-4745-b963-1551b2cf063a.png)
![image](https://user-images.githubusercontent.com/737603/227779515-e50c84c5-a2ae-46c4-bb7e-265aac8db23d.png)